File: frontend/src/types/report.ts
export interface ReportVO {
  id: number;
  reporterId: number;
  reporterName: string;
  targetType: string;
  targetId: number;
  reason: string;
  description: string | null;
  status: number;
  handlerId: number | null;
  handlerName: string | null;
  handleNote: string | null;
  createdAt: string;
  handledAt: string | null;
}

File: backend/src/main/java/com/campusforum/report/domain/Report.java
package com.campusforum.report.domain;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("reports")
public class Report {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long tenantId;
    private Long reporterId;
    private String targetType;
    private Long targetId;
    private String reason;
    private String description;
    private Integer status;
    private Long handlerId;
    private String handleNote;
    private LocalDateTime createdAt;
    private LocalDateTime handledAt;
}
